Tue Jun 15 15:02:28 2010  James Cameron  <quozl@us.netrek.org>

	* pptp.c (open_callmgr): fix usage of status returned by waitpid;
	it must be wrapped by WEXITSTATUS to shift bits as required.

--- pptp.c	2010-06-15 14:35:20.265852021 +0100
+++ pptp.c	2010-06-15 14:32:00.478100392 +0100
@@ -475,7 +475,7 @@
                 }
                 default: /* parent */
                     waitpid(pid, &status, 0);
-                    if (status!= 0)
+                    if (WEXITSTATUS(status) != 0)
                         fatal("Call manager exited with error %d", status);
                     break;
             }
